example-scenegraph-sdk/source/components/helloworld.brs

```diff
@@ -106,12 +106,137 @@ sub init()
     m.mparticle.setSessionAttribute("foo attribute key", "bar attribute value")
     m.mparticle.logCommerceEvent(productAction, promotionAction, impressions, {"foo-attribute":"bar-attribute-value"})
     
-    if APPInfo.IsDev() and args.RunTests = "true" and TF_Utils__IsFunction(TestRunner)
+    ' Custom Media Event
+    
+    customAttributes = {"example custom attribute" : "example custom attribute value"}
+    m.mparticle.logEvent("Custom Media Event", mparticleConstants().CUSTOM_EVENT_TYPE.MEDIA, customAttributes)
+    
+    ' Media API
+    customAttributes = {"example custom attribute" : "example custom attribute value"}    
+    mediaSession = mpConstants.MediaSession.build("ABC123", "Space Pilot 3000", mparticleConstants().MEDIA_CONTENT_TYPE.VIDEO, mparticleConstants().MEDIA_STREAM_TYPE.LIVE_STREAM, 1800000)
+    m.mparticle.media.logMediaSessionStart(mediaSession, customAttributes)
+    
+    print  ("Logging session after start: " + formatjson(mediaSession))
+    
+    segment = mpConstants.Segment.build("Chapter 1", 0, 183400)
+    mediaSession.segment = segment
+    m.mparticle.media.logSegmentStart(mediaSession, customAttributes)
+    
+    print  ("Logging session after segment start: " + formatjson(mediaSession))
+    
+    customAttributes = {"Source" : "Auto Playback"}    
+    m.mparticle.media.logPlay(mediaSession, customAttributes)
```

Comment:
As I mentioned on Slack, it's interesting that the api is `mparticle.media.methodName(mediaSession, options)`, whereas in our other media sdks it is mediaSession.methodName(options);. 

I am seeing it more clearly now vs when reviewing the PRs previously since you put example code here.  How much effort would it be to rework and match the other sdks api in this way?